docs/enhancements/template.md

```diff
@@ -26,6 +26,16 @@ To get started with this template:
 The `Metadata` section above is intended to support the creation of tooling
 around the enhancement process.
 
+Make sure that the enhancement covers the following scenarios:
+- REST API and Kubernetes API
+- Late-binding and bind-on-discovery
+- SNO and multinode
```

Comment:
should `compact cluster` be included also in this list? (for 3 master/worker nodes cluster)

Reply:
https://github.com/openshift/assisted-service/pull/4964#discussion_r1094299799